doc/syntax.bnf

```diff
@@ -17,7 +18,6 @@ reserved "let"
 reserved "open"
 reserved "print"
 reserved "private"
-reserved "proof"
 reserved "proofterm"
 reserved "protected"
 reserved "qed"
```

Comment:
comments from @barras and @fblanqui 

OK for ``begin`` but ``qed`` should then be replaced by ``end``

Reply:
fixed in 75c7b24